sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -290,8 +290,7 @@ case class DeleteFromTable(
  */
 case class UpdateTable(
     table: LogicalPlan,
-    columns: Seq[Expression],
-    values: Seq[Expression],
+    assignments: Seq[Assignment],
```

Comment:
Here we change the update set columns and values to `Seq[Assignment]`, to keep align with the `UpdateAction` in `MergeIntoTable`.